routes/camera.js

```diff
@@ -0,0 +1,117 @@
+import express from "express";
+const router = new express.Router();
+import http from "http";
+
+router.get("/save/:id(\\d\\d?)/", (req, res) => {
```

Comment:
Please send data using the body of the request (json?) instead of cluthering the url with it